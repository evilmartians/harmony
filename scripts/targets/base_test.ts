import { buildBasicPalette } from "./base.ts";
import { testExportTarget } from "./testUtils.ts";

Deno.test("base export", async (t) => {
  await testExportTarget(t, buildBasicPalette, [
    "index.js",
    "index.cjs",
    "index.d.ts",
  ]);
});
