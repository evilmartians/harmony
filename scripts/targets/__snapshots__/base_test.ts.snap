export const snapshot = {};

snapshot[`base export 1`] = `
'
  export default {
  "red": {
    "100": "oklch(0.966797 0.0171875 20)",
    "500": "oklch(0.742188 0.151562 20)"
  },
  "orange": {
    "100": "oklch(0.966797 0.0171875 43.3333)"
  }
};
'
`;

snapshot[`base export 2`] = `
\`'use strict';
module.exports = {
  "red": {
    "100": "oklch(0.966797 0.0171875 20)",
    "500": "oklch(0.742188 0.151562 20)"
  },
  "orange": {
    "100": "oklch(0.966797 0.0171875 43.3333)"
  }
};
Object.defineProperty(exports, '__esModule', { value: true });
\`
`;

snapshot[`base export 3`] = `
"declare namespace _default {
  let red: {
    100: string;
    500: string;
  };
  let orange: {
    100: string;
  };
}
export default _default;
"
`;
